cluster/images/kubemark/Makefile

```diff
@@ -22,15 +22,15 @@
 REGISTRY := $(if $(REGISTRY),$(REGISTRY),staging-k8s.gcr.io)
 IMAGE_TAG := $(if $(IMAGE_TAG),$(IMAGE_TAG),latest)
 
-all: gcloudpush
+all: dockerpush
 
 build:
 	docker build --pull -t $(REGISTRY)/kubemark:$(IMAGE_TAG) .
 
-gcloudpush: build
+dockerpush: build
```

Comment:
why not include this as an alias?
I disagree with this name, but this seems likely to break someone for minimal gain

Reply:
will revert, I agree